flask/testing.py

```diff
@@ -43,11 +44,34 @@ class FlaskClient(Client):
     information about how to use this class refer to
     :class:`werkzeug.test.Client`.
 
+    Uses default environment values
+
+    ::
+    
+        {
```

Comment:
Perhaps we should only say that we set some default env values, and that the exact values can be found at `client.environ_base`. Having them duplicated here may lead to one of the copies being out of date.